cluster/gce/util.sh

```diff
@@ -872,16 +853,8 @@ function construct-windows-kubelet-flags {
   # The directory where the TLS certs are located.
   flags+=" --cert-dir=${WINDOWS_PKI_DIR}"
 
-  flags+=" --network-plugin=cni"
-  flags+=" --cni-bin-dir=${WINDOWS_CNI_DIR}"
-  flags+=" --cni-conf-dir=${WINDOWS_CNI_CONFIG_DIR}"
   flags+=" --pod-manifest-path=${WINDOWS_MANIFESTS_DIR}"
 
-  # Windows images are large and we don't have gcr mirrors yet. Allow longer
-  # pull progress deadline.
-  flags+=" --image-pull-progress-deadline=5m"
-  flags+=" --enable-debugging-handlers=true"
```

Comment:
Why removing these lines?

Reply:
`--image-pull-progress-deadline` is removed as part of this PR:
https://github.com/kubernetes/kubernetes/blob/4a99a6eb12c7a7be6179b8eb3a0ec3bbeefadab7/pkg/kubelet/config/flags.go#L99-L100
`--enable-debugging-handlers=true` is the default:
https://github.com/kubernetes/kubernetes/blob/91b7fb4dc9f5d98583ee63be51dac7b6c0704b10/staging/src/k8s.io/kubelet/config/v1beta1/types.go#L296-L304